app/services/default_country.rb

```diff
@@ -0,0 +1,22 @@
+class DefaultCountry
+  def self.id
+    DefaultCountry.country.id
+  end
+
+  def self.code
+    DefaultCountry.country.iso
```

Comment:
I think we can do `self.country.iso` or even just `country.iso` here instead of `DefaultCountry.country.iso`? It feels a bit more idiomatic.

Reply:
IMO this fits better as instance state but that's a minor and subjective thing I guess. This would allow for

```
def code
  @country.iso
end
```

Reply:
I was thinking something similar, but we're not instanciating the `DefaultCountry` service here, right? It might need a bit of a rethink about how this service is used? 

We can probably do something really nice here with caching so that we're not hitting the database all the time to fetch these values that almost never change, but it might need a bit of careful thinking (and a separate PR).

Those kinds of options are at least possible now that this service is in place, which is great :tada:

Reply:
> Those kinds of options are at least possible now that this service is in place, which is great tada

definitely, this gives us the chance to not overthink things now and iterate on this.

What you say suggest sounds good. It's as simple as storing a DefaultCountry instance in Rails.config setting of our own at boot-time and it'll be loaded into memory during runtime. It's a problem all `Spree::Config` settings face.